react/__tests__/components/__snapshots__/SearchBar.test.js.snap

```diff
@@ -24,6 +24,7 @@ exports[`<SearchBar /> should match snapshot 1`] = `
               id="downshift-1-input"
               placeholder="Search"
               role="combobox"
+              runtime="[object Object]"
```

Comment:
What is this snapshot useful for?

Reply:
It tests if there was any structure change to the component.

Reply:
Testing if it contains `object Object` doesn't seem like a useful test for me—quite the opposite, it bricks the evolution of the component with an assertion that does not sums up any confidence to whether the component is working as expected or not.

Reply:
I uderstand, but since the snapshot is automatically generated and my mocks only states that runtime is a HOC that returns a component i don't know what else i can do to improve it. If you have any suggestion i will be glad to hear.